### PR TITLE
feat: Add responsiveness in all pages

### PR DESCRIPTION
**Pull request by jalajcodes**
https://deploy-preview-5--max-store.netlify.app/

https://user-images.githubusercontent.com/17108695/154834730-58734c25-b01c-4516-b655-f1e32989bb95.mp4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,18 +18,16 @@
...
-            <img src="./images/logo.png" alt="logo" width="200" />
-          </div>
-        </a>
-        <div class="nav__search">
-          <input type="text" placeholder="Search" />
-          <button>
-            <i class="fas fa-search"></i>
-          </button>
+        <div class="nav__logo">
+          <img src="./images/logo.png" alt="logo" width="200" />
```
Comment (MayankKumar10) [inline index.html]: try to avoid using inline css

File: index.html [hunk 2]
```diff
@@ -71,7 +72,7 @@ <h1>All You Need for Your <span>Wardrobe</span></h1>
 
     <section class="categories container" id="categories">
       <h2>Categories</h2>
-      <div class="grid col-4">
+      <div class="categories__container grid col-autofit">
         <div class="category">
           <img src="https://picsum.photos/500/?random=1" alt="" />
```
Comment (MayankKumar10) [inline index.html]: add value to alt tag

File: styles/cart.scss [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
-  flex-direction: ROW;
   gap: 2rem;
 
+  @media screen and (max-width: 768px) {
+    flex-direction: column;
+  }
+
   &__price-info {
     flex: 0 0 30%;
     box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.1);
```
Comment (MayankKumar10) [inline styles/cart.scss:11]: If you used this box-shadow property elsewhere make it a helper class

File: styles/index.scss [hunk 4]
```diff
@@ -20,6 +20,7 @@ h2 {
 .container {
   max-width: 1280px;
```
Comment (MayankKumar10) [inline styles/index.scss]: Use rem instead of px

**Review by MayankKumar10 — APPROVED** [review]
Look into these everything looks good to me

**Review by venkateshwarans — APPROVED** [review]
Everything looks good to me. Great work on the responsiveness and scss. One thing I would nitpick is - Categories and Products cards are not aligned - end to end. 
Great work nonetheless.

**Comment by jalajcodes** [comment]
> Everything looks good to me. Great work on the responsiveness and scss. One thing I would nitpick is - Categories and Products cards are not aligned - end to end. Great work nonetheless.

Fixed it

**Pull request merged**